modules/fedimint-ln-server/src/lib.rs

```diff
@@ -85,7 +85,19 @@ lazy_static! {
         "contracts::FundedContract::Outgoing"
     ))
     .unwrap();
-    static ref AMOUNTS_BUCKETS_SATS: Vec<f64> = vec![0.0, 0.5, 1.0, 1000.0];
+    static ref AMOUNTS_BUCKETS_SATS: Vec<f64> = vec![
+        0.0,
```

Comment:
Isn't 0.0 useless?

Reply:
It's one way to track what's zero.

Reply:
One example where the zero makes it clear:
```
mint_redeemed_ecash_sats_bucket{le="0"} 0
mint_redeemed_ecash_sats_bucket{le="0.5"} 678
mint_redeemed_ecash_sats_bucket{le="1"} 722
```

```
mint_redeemed_ecash_fees_sats_bucket{le="0"} 756
mint_redeemed_ecash_fees_sats_bucket{le="0.5"} 756
mint_redeemed_ecash_fees_sats_bucket{le="1"} 756
```

Reply:
I don't get it. We don't even have fees for ecash txes, do we? And peg-in/peg-outs must have non-zero fees?

Reply:
It seems we do have fees for ecash transactions.

Reply:
> It seems we do have fees for ecash transactions

WAT? How? Why I don't know about it.

Reply:
https://github.com/fedimint/fedimint/blob/b013367ba68474c89e213d439a8c747d31acddf5/modules/fedimint-mint-server/src/lib.rs#L351

Reply:
TIL.

Reply:
Yep, every module *could* charge fees in theory. In practice we don't do it since it would massively complicate the client (especially for e-cash). It would be easy to get scenarios where coin selection runs into loops where adding/removing e-cash makes adding/removing more e-cash necessary because of fees.

Note: these fees are a separate concept from withdraw or LN fees, they'd be going to guardians (well, technically the federation would just become overcollateralized since there is no way to withdraw them).

Reply:
> It would be easy to get scenarios where coin selection runs into loops where adding/removing e-cash makes adding/removing more e-cash necessary because of fees.

I don't think hitting this would be common. It is exactly like in Bitcoin - not really a problem people complain about. A bit of looping, and tiny overpaying fees should do.

IMO, the bigger issue the whole new aspect of "guardians making money". We have no affordances for it, no means for them to do anything useful with it (withdraw or e.g. contribute to peg-outs, or something), etc.